README.md

```diff
@@ -39,6 +39,8 @@ On Linux, you may install the .deb file, then run `/opt/rcv/bin/RCTab` to launch
     
     If the expected version isn't returned, you'll need to follow the instructions [here](https://www.java.com/en/download/help/path.xml) on how to set your Java path.
 
+    If you need to regularly switch between Java versions, consider installing [jEnv](https://www.jenv.be/). For a list of the Java versions installed on your machine, run `/usr/libexec/java_home -V` on MacOS or `update-alternatives --config java` on Linux.
```

Comment:
```suggestion
    If you are using Linux or MacOS and need to regularly switch between Java versions, consider installing [jEnv](https://www.jenv.be/). For a list of the Java versions installed on your machine, run `/usr/libexec/java_home -V` on MacOS or `update-alternatives --config java` on Linux.
```

Reply:
I started looking for how to check my version on windows when I realized jEnv is only built for MacOS and Linux